code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -36,6 +36,10 @@
 		if(initial(uses) > 1) //no need to tell 'em if it was one-use anyway!
 			to_chat(owner, "<span class='warning'>[name] has run out of uses!</span>")
 		qdel(src)
+		if(istype(src, /datum/action/innate/ai/overload_machine))
+			usr.verbs -= /mob/living/silicon/ai/proc/overload_machine
+		else if(istype(src, /datum/action/innate/ai/override_machine))

```

Comment:
Ditto

Reply:
I already had deletion code in the proc itself for when it ran out of charges, what was I thinking..